React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -126,6 +126,10 @@ - (void)presentViewController:(UIViewController *)alertController
 
   NSInteger index = 0;
   bool isCancelButtonIndex = false;
+  // The handler for a button might get called more than once when tapping outside
+  // the action sheet on iPad. RCTResponseSenderBlock can only be called once so
+  // keep track of callback invocation here.
+  __block bool callbackInvoked = false;
```

Comment:
this is never set to true? the condition in L146 will always pass. am i missing something obvious?

Reply:
Oh you're right, my bad I removed it to test before / after and forgot to put it back -_-